examples/neural_compressor/language-modeling/run_clm.py

```diff
@@ -611,10 +611,15 @@ def train_func(model):
 
             q8_config.set_config("model.framework", "pytorch_fx")
 
-        calib_dataloader = trainer.get_train_dataloader() if quant_approach != IncQuantizationMode.DYNAMIC else None
-        quantizer = IncQuantizer(
-            q8_config, eval_func=eval_func, train_func=train_func, calib_dataloader=calib_dataloader
-        )
+        if quant_approach == IncQuantizationMode.STATIC:
+            calib_dataloader = trainer.get_train_dataloader()
+            quantizer = IncQuantizer(q8_config, eval_func=eval_func, calib_dataloader=calib_dataloader)
+        else:
+            example_inputs = next(iter(trainer.get_train_dataloader()))
+            example_inputs = [example_inputs[key] for key in example_inputs]
+            quantizer = IncQuantizer(
+                q8_config, eval_func=eval_func, train_func=train_func, example_inputs=example_inputs
+            )

```

Comment:
Why giving an `example_inputs` in the case of dynamic quantization ? 
I would suggest :
```suggestion
        example_inputs = None
        if quant_approach == IncQuantizationMode.AWARE_TRAINING:
            example_inputs = next(iter(trainer.get_train_dataloader()))
            example_inputs = [example_inputs[key] for key in example_inputs]
        calib_dataloader = trainer.get_train_dataloader() if quant_approach == IncQuantizationMode.STATIC else None
        quantizer = IncQuantizer(
            q8_config,
            eval_func=eval_func,
            calib_dataloader=calib_dataloader,
            example_inputs=example_inputs,
        )
```
Also, what is the interest of having multiple inputs in `example_inputs` instead of using only one ?